src/vs/workbench/contrib/positronPlots/browser/positronPlotsService.ts

```diff
@@ -1492,6 +1492,8 @@ export class PositronPlotsService extends Disposable implements IPositronPlotsSe
 	}
 
 	private createRuntimePlotClient(comm: PositronPlotCommProxy, metadata: IPositronPlotMetadata, location: PlotClientLocation = PlotClientLocation.View) {
+		// for Python plots, use intrinsic sizing policy as default
+		this._selectedSizingPolicy = metadata.session_id.startsWith('python') ? this._intrinsicSizingPolicy : this._selectedSizingPolicy;
```

Comment:
Oh, I actually do have one suggestion. This does work for now, but it isn't guaranteed that the `session_id` will be named in this way moving forward. Can we get more stable access to the session's language here?

Reply:
Yeah! Let me plumb through a `languageId` on the metadata 👍